### PR TITLE
feat: SetSnsTokenSwapOpenTimeWindow request/response converters

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Add SetSnsTokenSwapOpenTimeWindow support

# Changes

- type SetSnsTokenSwapOpenTimeWindow
- request converter function
- response converter function

### CODE REVIEW

File: packages/nns/src/canisters/governance/request.converters.ts [hunk 1]
```diff
@@ -195,6 +195,37 @@ const fromAction = (action: Action): RawAction => {
     };
   }
 
+  if ("SetSnsTokenSwapOpenTimeWindow" in action) {
+    const swapOpenTimeWindow = action.SetSnsTokenSwapOpenTimeWindow;
```
Comment (peterpeterparker) [inline packages/nns/src/canisters/governance/request.converters.ts:199]: nth: deconstruct `{request, swapCanisterId}`

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks!

**Comment by peterpeterparker** [comment]
@mstrasinskis can you add an entry in changelog?

File: packages/nns/src/canisters/governance/request.converters.ts (hunk 1 above)
Comment (mstrasinskis) [reply packages/nns/src/canisters/governance/request.converters.ts]: Good point. Done.

**Pull request merged**